assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js

```diff
@@ -113,23 +113,14 @@ export default function Footer( {
 		select( CORE_MODULES ).isModuleConnected( 'analytics-4' )
 	);
 
-	const saveError = useSelect( ( select ) => {
-		if ( haveSettingsChanged && selectedMetrics?.length < 2 ) {
-			return {
-				message: __( 'Select at least 2 metrics', 'google-site-kit' ),
-			};
-		}
-
-		return select( CORE_USER ).getErrorForAction(
-			'saveKeyMetricsSettings',
-			[
-				{
-					...keyMetricsSettings,
-					widgetSlugs: selectedMetrics,
-				},
-			]
-		);
-	} );
+	const APIError = useSelect( ( select ) =>
```

Comment:
Let's keep the variable name `saveError` as I think that relates more to what the error is about, and also the casing for `APIError` is slightly different from our usual conventions.